src/main/java/ru/jma/userservice/model/User.java

```diff
@@ -0,0 +1,17 @@
+package ru.jma.userservice.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "usr")
+public class User {
```

Comment:
Create BasicEntity with one field objectId annotated with @MappedSuperClass
Change table name to "users"
Use Lombok for getters/setter, equals&hashcode with callsuper = true

Reply:
@Entity
@Table(name = "users")
@Data
@NoArgsConstructor
@EqualsAndHashcode(callsuper = true)
public class User extends BaseEnity